src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -768,6 +769,13 @@ public RequestSpecification getMultipleAuthHeadersExternal(String role, String f
         return getMultipleAuthHeaders(bearerTokenForSuperUser);
     }
 
+    public RequestSpecification getMultipleAuthHeadersExternalForSuperUser(List<String> roles, String firstName,
+                                                                           String lastName, String email) {
+        String bearerTokenForSuperUser = idamOpenIdClient.getExternalOpenIdToken(roles, firstName, lastName, email);
+        log.info("SuperUser Token:" + bearerTokenForSuperUser);

```

Comment:
Do we need to log this?

Reply:
need to verify the logs if in case fails the functional test cases after that will remove it ..